test/functional/rpc_scantxoutset.py

```diff
@@ -20,47 +24,42 @@ def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
 
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
+    def sendtodestination(self, destination, amount):
+        # interpret strings as addresses, assume scriptPubKey otherwise
+        if isinstance(destination, str):
+            destination = address_to_scriptpubkey(destination)
```

Comment:
last commit:

An alternative would be `self.nodes[0].validateaddress(destination)['scriptPubKey']`, but both work fine.

Reply:
Ah nice, for some reason I thought `getaddressinfo` is the only RPC that can do that. Decided to keep as it is though, to need fewer RPC calls (can be changed if other reviewers prefer to call `validateaddress`).